packages/csharp/PortingAssistant/PortingAssistant.Telemetry/Models/MetricsBase.cs

```diff
@@ -12,5 +14,25 @@ public class MetricsBase
         public string PortingAssistantVersion { get; set; }
         public string TargetFramework { get; set; }
         public string TimeStamp { get; set; }
+
+        /// <summary>
+        /// This property uniquely identifies the customers using porting assistant 
+        /// Auto populate this field for all the metric logs
+        /// </summary>
+        private string _customerID;
+        public string CustomerID
+        {
+            get
+            {
+                if (string.IsNullOrWhiteSpace(_customerID))
+                {
+                    _customerID = NetworkInterface.GetAllNetworkInterfaces()
+                                         .Where(nic => nic.OperationalStatus == OperationalStatus.Up && nic.NetworkInterfaceType != NetworkInterfaceType.Loopback)
+                                         .Select(nic => nic.GetPhysicalAddress().ToString())
+                                         .FirstOrDefault();
```

Comment:
We probably need to generate our own Id instead of this

Reply:
Maybe hash of the physical address?

Reply:
we were discussing to generate a guid and store it in the config.json and use it to identity the customer